docs/kb/semgrep-ci/azure-using-templates-with-semgrep.md

```diff
@@ -0,0 +1,75 @@
+---
+tags:
+  - Azure
+  - Templates
+description: Using templates in Azure pipelines with Semgrep commands.
```

Comment:
```suggestion
description: Running Semgrep commands through templates in Azure Pipelines.
```
Tried to make this clearer. The noun "with Semgrep commands" could mean we are executing Semgrep commands on the Azure Pipeline, however weird that sounds to us, someone who isn't too familiar with Semgrep might think we're scanning the YAML file.